core/src/main/java/io/confluent/rest/exceptions/SuppressedJsonParseExceptionMapper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.rest.exceptions;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class SuppressedJsonParseExceptionMapper implements ExceptionMapper<JsonParseException> {
+  public SuppressedJsonParseExceptionMapper() {}
+
+  public Response toResponse(JsonParseException exception) {
+    return Response.status(Status.BAD_REQUEST)
+        .entity("Error parsing JSON")
```

Comment:
Can't we at least send exception.getMessage()? The user is losing critical information to understand what went wrong.

Reply:
cc @srpanwar-confluent